datasync/chaindatafetcher/kas/contract_caller_test.go

```diff
@@ -58,62 +58,58 @@ func (s *SuiteContractCaller) TearDownTest() {
 	s.ctrl.Finish()
 }
 
-func setExpectation(m *mocks.MockBlockchainAPI, contract *common.Address, blockNumber *big.Int, data, result []byte) {
+func setExpectation(m *mocks.MockBlockchainAPI, contract *common.Address, data, result []byte) {
 	arg := api.CallArgs{
 		From: common.Address{},
 		To:   contract,
 		Data: data,
 	}
 
-	m.EXPECT().Call(gomock.Any(), gomock.Eq(arg), gomock.Eq(rpc.BlockNumber(blockNumber.Int64()))).Return(result, nil).Times(1)
+	m.EXPECT().Call(gomock.Any(), gomock.Eq(arg), gomock.Eq(rpc.LatestBlockNumber)).Return(result, nil).Times(1)
 }
 
 func (s *SuiteContractCaller) TestContractCaller_IsKIP13_Success() {
 	addr := common.HexToAddress("1")
 	s.True(addr != (common.Address{})) // make sure that the address is not empty address
-	blockNumber := big.NewInt(1)
 
 	// KIP13 validation
-	setExpectation(s.api, &addr, blockNumber, ikip13Input, decodedTrue)
-	setExpectation(s.api, &addr, blockNumber, ikip13InvalidInput, decodedFalse)
+	setExpectation(s.api, &addr, ikip13Input, decodedTrue)
+	setExpectation(s.api, &addr, ikip13InvalidInput, decodedFalse)
 
-	isKIP13, err := s.caller.isKIP13(addr, blockNumber)
+	isKIP13, err := s.caller.isKIP13(addr, nil)
```

Comment:
If we change this, we don't need to change `datasync/chaindatafetcher/kas/contract_caller.go`. 
```suggestion
	isKIP13, err := s.caller.isKIP13(addr, rpc.LatestBlockNumber)
```

Reply:
The type of parameter is `*big.Int` and it will convert to `rpc.BlockNumber` in `CodeAt` and `CallContract`.